### ISSUE TITLE
Coveralls 422 에러

### ISSUE THREAD

**Issue by hy00nc** [swsnu/swpp2020-team6#51]
안녕하세요, Coveralls 관련 업데이트 드립니다.
어제 오늘 여러 방법을 시도해봤는데요, 결국 [Coveralls의 문제](https://github.com/lemurheavy/coveralls-public/issues/632)라는 결론을 내렸습니다.. (master commit log 더럽혀서 대단히 죄송합니다ㅜ)

public repo라서 token없이도 repo를 찾을 수 있어야 하는데 못 찾네요.
어제 그래서 Coveralls repo 리스트에서 제거 후 다시 추가했는데도 작동을 안 합니다😭  (어제 오늘 추가해드린 다른 팀들은 되는걸로 봐서 복불복인 거 같습니다..)
일단 .travis.yml은 지금 snapshot이 문제 없는 버전이니, 빌드 계속 진행하면서 경과를 봐야할 것 같습니다.
Test coverage 채점은 일단 travis log에서 직접 확인하든 해서 진행할테니 걱정 않으셔도 됩니다.

**Comment by ddanddan18** [comment]
네 알겠습니다! 안그래도 원래 local 브랜치에서 토큰과 같이 coveralls 하는 방법이 통했었는데, 오늘은 그것도 안되길래 걱정하고 있었습니다.. 아니면 저희가 sprint3 태그 달 때 커버리지 관련 스크린샷이라도 첨부하도록 하겠습니다!

**Comment by hy00nc** [comment]
아마 로컬에서 안 됐던 이유는 제가 새로 레포를 추가해서 토큰이 renew되어서 그런 것 같습니다.
말씀해주신 방법으로 했더니 저도 됐네요 🤔 
그리고 저번에 말씀해주신 backend 결과만 나오는 것에 대해서는 뭔가 coveralls에서 마지막 build인 backend build 기준으로만 업데이트가 되는 것 같습니다. 두 레포트를 merge 해서 보내야 할 거 같은데 업데이트 생기면 알려드리겠습니다.

**Comment by ddanddan18** [comment]
넵
local에서 yarn package coveralls 를 실행하면 (아마 토큰관련 때문인지) 아예 멈춰있어서,
그냥 pip coveralls 만 token 설정 후 업데이트 하곤 했는데, 

travis-ci에서 422 에러가 안뜬다면 frontend, backend 결과 둘 다 나올 수 있을 것 같습니다!

**Comment by hy00nc** [comment]
좋습니다!! 일단 frontend 랑 backend 합쳐서 보내게 업데이트는 해두었습니다. 아직 422 에러가 나긴해요 ㅜㅜ 수고하세요!
